lib/HTFeed/Stage/Pack.pm

```diff
@@ -104,11 +104,13 @@ sub zip{
         }
     }
 
-    # Get size after zipping and report size delta as a job metric for Pack.
-    my $end_size = -s $zip_path;
-    my $delta_size = $start_size - $end_size;
-    $self->{job_metrics}->add("ingest_pack_bytes", $delta_size);
-
+    my $end_time = $self->{job_metrics}->time;
+    my $delta_time = $end_time - $start_time;
+    $self->{job_metrics}->add("ingest_pack_seconds_total", $delta_time);
+    my $bytes_w = -s $zip_path;
+    $self->{job_metrics}->add("ingest_pack_bytes_r_total", $bytes_r);
```

Comment:
Here we have `r` vs. `w` that is the minimal difference between the metrics so definitely does help differentiate/clarify